### DISCUSSION TITLE
Target Rate: shift the 24h period start

### DISCUSSION THREAD

**Discussion by slip83** [BottlecapDave/HomeAssistant-OctopusEnergy#601] [Ideas]
As the most expensive rate is most likely around 6pm and the latest publishing time of next day rates is 7pm at latest would it be possible to shift the start of the target rate period? 
It's very often that I miss 2/3 slots of cheap rates between 10pm and midnight which inturn makes the charging slightly more expensive... If we evaluate target rates between 8pm and 8pm the next day we could again shave some precious pounds of our bills.

**Comment by BottlecapDave** [comment]
Hello. When you setup a target rate sensor, you can set the evaluation time as per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/setup_target_rate.md#target-timeframe). If you update the sensor and set both from/to to 20:00 then it'll evaluate times between 8pm on one day and 8pm on the next.

**Comment by slip83** [comment]
Ah, that's what I was doing wrong then! I only tried after 8pm but it was stopping at midnight... Thanks!
